jetty-core/jetty-siwe/src/main/java/module-info.java

```diff
@@ -0,0 +1,20 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+module org.eclipse.jetty.siwe
```

Comment:
Your pom.xml says you are exporting a ServiceLoader service, but I don't see it declared in here?

Reply:
good catch, added it now